tests/Resources/Common.robot

```diff
@@ -15,7 +15,7 @@ Begin Web Test
     Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
     Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
     Wait for RHODS Dashboard to Load
-    Launch JupyterHub From RHODS Dashboard Dropdown
+    Launch JupyterHub From RHODS Dashboard Link
```

Comment:
In order to maintain compatibility of tests with the current version in production this changes should be done like this:

```
${version-check} =  Is RHODS Version Greater Or Equal Than  1.4.0
IF  ${version-check}==True
  Launch JupyterHub From RHODS Dashboard Link
ELSE
  Launch JupyterHub From RHODS Dashboard Dropdown
END
```

Reply:
+1, need to maintain compatibility

Reply:
Done!